src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -495,7 +500,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
     ];
 
     const results = new SearchResults(helper.state, [
-      { hits, queryID: 'theQueryID' },
+      createAlgoliaResponse({ hits, queryID: 'theQueryID' }),
```

Comment:
TS also fails here, but fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36859